flang/include/flang/Evaluate/intrinsics.h

```diff
@@ -76,6 +76,9 @@ class IntrinsicProcTable {
   // Inquiry intrinsics are defined in section 16.7, table 16.1
   IntrinsicClass GetIntrinsicClass(const std::string &) const;
 
+  // Return the generic name of a specific intrinsic name.
```

Comment:
It was unclear what happens when `name` is not an intrinsic name or cannot be converted to the "generic" name when I first read this. (But I could infer from the implementation just below.)

Reply:
Good point, I have updated the comment to tell that "name" is returned if it is not known to be an intrinsic.